[git] add default configured remote for git pull & git push

vince-fugnitto opened the pull request:
Fixes #2195

- Added the commands `Git: Pull` and `Git: Push` which use the default configured remote
- Re-named the command `Git: Pull...` to `Git: Pull from...` to align
- Re-named the command `Git: Push...` to `Git: Push to...` to align

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>

<!-- Please provide a clear and meaningful description to the CHANGELOG.md file if this PR contributes some significant changes -->

elaihau commented on packages/git/src/browser/git-quick-open-service.ts:
```diff
@@ -133,6 +146,19 @@ export class GitQuickOpenService {
+        const remote = await this.getRemotes();
+        const defaultRemote = remote[0];
+        if (repository) {
+            try {
```
this is a duplication of lines 179-183, is it possible to wrap it up in a separate function to reuse?

vince-fugnitto commented:
@elaihau I've updated the PR based on your comments, and also added some logging to the new commands since the git actions were a little too silent for my taste.
